NIPTool/API/external/api/api_v1/endpoints/update.py

```diff
@@ -1,26 +1,51 @@
 import logging
-from datetime import datetime
 from typing import Iterable
+from typing import Optional
 
 from fastapi import APIRouter, Depends, Request
 from fastapi.responses import RedirectResponse
 from NIPTool.adapter.plugin import NiptAdapter
 from NIPTool.crud import find, update
-from NIPTool.models.database import Sample, User
-from NIPTool.server.web_app.api.deps import get_nipt_adapter
-from NIPTool.server.web_app.utils import *
+
+from NIPTool.models.database import User
+from NIPTool.API.external.utils import *
+from NIPTool.API.external.api.deps import get_nipt_adapter, get_current_active_user
+from datetime import datetime
 
 router = APIRouter()
 
 LOG = logging.getLogger(__name__)
 
+from pydantic import BaseModel
+
+
+class Item(BaseModel):
+    form_id: str
+    sample_id: Optional[str]
+
+
+@router.post("/update_debugging")
+def update_debugging(
+    request: Request, item: Item, adapter: NiptAdapter = Depends(get_nipt_adapter)
+):
+    """Update the database"""
+    print(item)
+    print("hej")
```

Comment:
Keep?

Reply:
Lets keep. its for debugging and will be fixed in separate pr. Was my intention